db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_08_25_093845) do
+ActiveRecord::Schema.define(version: 2022_08_27_064529) do
```

Comment:
why this was changed?

Reply:
I did a migration after pool main branch. I suppose it has made this change